### ISSUE TITLE
NGFF Testing

### ISSUE THREAD

**Issue by dgault** [ome/ZarrReader#67]
This is a placeholder issue for reports on testing of NGFF IDR datasets

**Comment by jburel** [comment]
image: 10648053
<img width="1512" alt="idr-test-10648053" src="https://github.com/ome/ZarrReader/assets/1022396/a53856d5-0a85-4c73-89d9-8e1b9cbd4488">
<img width="1512" alt="idr-10648053" src="https://github.com/ome/ZarrReader/assets/1022396/808062d5-6e9a-4a14-89cb-d97421d2930f">
lower bound indicated as NaN

**Comment by will-moore** [comment]
That issue is not related to NGFF data but is due to float/double pixel type.
Fixed at https://github.com/ome/omero-iviewer/pull/460

**Comment by jburel** [comment]
Exploring images with z section e.g. https://idr-testing.openmicroscopy.org/webclient/img_detail/2857886/
The loading of the planes is slow when scrolling through z (I assume the same is true for t) this means that the user will move the slider knob and won't see any change

**Comment by jburel** [comment]
https://idr-testing.openmicroscopy.org/webclient/img_detail/3193545/ missing thumbnails for some fields
<img width="1512" alt="Screenshot 2023-11-10 at 10 12 04" src="https://github.com/ome/ZarrReader/assets/1022396/82b58866-77a8-4e54-b866-6398a5c22d14">

**Comment by jburel** [comment]
See also https://idr-testing.openmicroscopy.org/webclient/?show=dataset-13751
<img width="1512" alt="Screenshot 2023-11-21 at 07 59 48" src="https://github.com/ome/ZarrReader/assets/1022396/6931f735-6cb2-4978-8408-aef7af4cf3bd">

**Comment by jburel** [comment]
Same https://idr-testing.openmicroscopy.org/webclient/?show=dataset-13751
<img width="1512" alt="Screenshot 2023-11-21 at 08 02 02" src="https://github.com/ome/ZarrReader/assets/1022396/e3fe8585-080b-4e08-9b72-297ff0960f42">

**Comment by jburel** [comment]
https://idr-testing.openmicroscopy.org/webclient/img_detail/12555350/
<img width="1512" alt="Screenshot 2023-11-21 at 08 04 49" src="https://github.com/ome/ZarrReader/assets/1022396/20df6e87-fd62-4b3f-a680-0bbadc26c5af">

**Comment by will-moore** [comment]
@jburel I think what has happened in https://github.com/ome/ZarrReader/issues/67#issuecomment-1820363750 is that I've saved new rendering settings to all images in the Plate while logged-in as demo user with "Save to All" in the webclient, but that has only requested new thumbnails for the first field of each Well. Now, when you view the images as public user, I think you don't have permissions to generate new thumbnails.

The reason I used "Save to All" as demo user was that this was a good test to check every (first field) Image in the Plate was valid, by regenerating thumbnails (and checking that they were the same as the thumbnails they replaced). However, this won't happen when we do this "for real" on IDR next.
